man/fi_msg.3.md

```diff
@@ -138,12 +138,12 @@ per call through the use of flags.  The fi_sendmsg function takes a
 
 {% highlight c %}
 struct fi_msg {
-	const struct iovec *msg_iov;/* scatter-gather array */
-	void               **desc;  /* local request descriptor */
-	size_t             iov_count;/* # elements in iov */
-	const void         *addr;   /* optional endpoint address */
-	void               *context;/* user-defined context */
-	uint64_t           data;    /* optional message data */
+        const struct iovec *msg_iov;  /* scatter-gather array */
+        void               **desc;    /* local request descriptors */
+        size_t             iov_count; /* # elements in iov */
+        fi_addr_t          addr;      /* optional endpoint address */
```

Comment:
I'm not sure why GitHub is rendering the diff like this.  I just tidied up the alignment while I was fixing the incorrect type.  That included converting to all-spaces for indentation/alignment.  If you would prefer that the alignment remain slightly jagged and partially tab-based, I can put it back.  The whitespace choices don't match the raw source in either case (it's 100% tabs with no comments in fi_endpoint.h).